test/Microsoft.AspNetCore.Server.KestrelTests/BadHttpRequestTests.cs

```diff
@@ -82,6 +107,11 @@ public class BadHttpRequestTests
         [InlineData("GET / HTTP/1.\r\n")]
         [InlineData("GET / hello\r\n")]
         [InlineData("GET / 8charact\r\n")]
+        [InlineData("GET /  HTTP/1.1\r\n")]
+        [InlineData("GET / / HTTP/1.1\r\n")]
+        [InlineData("GET http://  HTTP/1.1\r\n")]
+        [InlineData("GET https://  HTTP/1.1\r\n")]
+        [InlineData("GET https:// / HTTP/1.1\r\n")]
```

Comment:
These are here because there is a space or a slash where the HTTP version should begin.